pkg/cmd/cli/cmd/login/helpers.go

```diff
@@ -108,8 +110,25 @@ func getHostPort(hostURL string) (string, string, *url.URL, error) {
 	return host, port, parsedURL, err
 }
 
-func whoAmI(client *client.Client) (*api.User, error) {
+func whoAmI(clientConfig *restclient.Config) (*api.User, error) {
+	client, err := client.New(clientConfig)
+
 	me, err := client.Users().Get("~")
+
+	// if we're talking to kube (or likely talking to kube),
```

Comment:
Ah, I had read the comment as if the wording inside the parenthesis was redundant, or maybe accidentally left there after rephrasing the entire sentence, but it made more sense after reading [this comment](https://github.com/openshift/origin/pull/11120/files#diff-823ab4e35468c6cb31f1c30d0e3c41a2R266). Please ignore my first comment, it's fine the way it is